ds-rshiny/files/docker/Dockerfile

```diff
@@ -35,25 +36,30 @@ RUN apt-get update && \
         libxt-dev \
         pandoc \
         pandoc-citeproc \
+        libssl-dev && \
+    apt-get install -yqq \
         r-base \
-        r-base-dev \
-        r-cran-rjava
+        r-base-core \
+        r-recommended \
+        r-base-dev
 
 # Install Shiny
 RUN R -e 'install.packages(c("shiny", "rmarkdown"), repos="https://cran.uni-muenster.de/")'
 
 # Install app dependencies and fail if any missing
-RUN R -e 'packages = c("RJDBC", "shinythemes", "ggplot2", "reshape2"); \
+RUN R -e 'packages = c("shinythemes", "ggplot2", "reshape2"); \
     install.packages(packages, repos="http://cran.uni-muenster.de/"); \
     package.check <- lapply(packages, FUN = function(p) { if(!require(p, character.only = TRUE)) { stop("package not found!") } })'
 
 # Cleanup apt
-RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apt-get autoremove --purge && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/archives
```

Comment:
This is a bit pointless actually. Given container images are layers, this does not actually make the image any smaller. I see it's already been like that before though. I'd either remove it, or, if possible, add it to the layer which runs `apt-get install`.

Reply:
Makes sense! on the other side I see it is more to ensure that later on noone can make use of apt.

Will give it a try and come back! 👍

Reply:
Whether one can use `apt` is more a matter of permissions - if I'm not mistaken, even on OpenShift in building you can switch back to root and then do `update` and `install` all over again ;)

Reply:
ok, I did some refactor with success and it really looks much more better! thanks for taking a look @michaelsauter !! 🚀